recipe/meta.yaml

```
@@ -44,11 +44,12 @@ requirements:
     - libuv 1.44    # [not win]
     - openssl 1.1.1 # [not win]
     - zlib 1.2      # [not win]
+    - icu {{ icu }}
   run:
     # icu min version is 69. It won't compile without this minumum version.
     # until the distribution gets updated to a new icu, internationalization is disabled. 
-    - icu >=58,<59.0a0        # [not (win or s390x or aarch64 or (osx and arm64))]
-    - icu >=68,<69.0a0      # [s390x or aarch64 or (osx and arm64)]
+    - icu >=58,<=73.1      # [not (win or s390x or aarch64 or (osx and arm64))]
+    - icu >=68,<=73.1      # [s390x or aarch64 or (osx and arm64)]
```

```suggestion
    - {{ pin_compatible('icu') }}
```
This should replace line 51/52. This results in icu >=73.1,<74.0a0